src/marketplace-restserver/build/marketplace-restserver.k8s.dockerfile

```diff
@@ -1,4 +1,4 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 
-FROM docker.io/openpai/pai-marketplace-restserver
+FROM docker.io/openpai/pai-marketplace-restserver:v1.5.0
```

Comment:
is marketplace v1.5.0 released ?

Reply:
it has been released

Reply:
> it has been released

Thx, pls remember to sync the changes to master branch